src/Middleware/NodeServices/src/HostingModels/HttpNodeInstance.cs

```diff
@@ -19,6 +22,7 @@ namespace Microsoft.AspNetCore.NodeServices.HostingModels
     /// accept RPC invocations.
     /// </summary>
     /// <seealso cref="Microsoft.AspNetCore.NodeServices.HostingModels.OutOfProcessNodeInstance" />
+    [Obsolete("Use Microsoft.AspNetCore.SpaServices.Extensions")]
```

Comment:
It's `internal` anyway

Reply:
Everywhere I've made internal types obsolete it's because they reference obsolete types and I figured it was cleaner and more accurate than line pragmas.

Reply:
OK, whatever works best.